resources/mediaplayer/src/test/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackImplTest.java

```diff
@@ -50,32 +66,104 @@ public static void tearDownClass() throws Exception {
     }
 
     @Before
-    public void setUp() {
+    public void setUp() throws IOException, UnsupportedAudioFileException {
+        url_pcm = WavTrackImplTest.class.getClassLoader().getResource("welcome.wav");
+        url_ulaw = WavTrackImplTest.class.getClassLoader().getResource("welcome_ulaw.wav");
+        url_alaw = WavTrackImplTest.class.getClassLoader().getResource("welcome_alaw.wav");
+        wavTrackPcm = new WavTrackImpl(url_pcm);
+        wavTrackUlaw = new WavTrackImpl(url_ulaw);
+        wavTrackAlaw = new WavTrackImpl(url_alaw);
     }
 
     @After
     public void tearDown() {
+        wavTrackPcm.close();
+        wavTrackUlaw.close();
+        wavTrackAlaw.close();
     }
 
     /**
      * Test of setPeriod method, of class WavTrackImpl.
      */
     @Test
     public void testDurationd() throws Exception {
```

Comment:
Needs to be renamed (remove extra "d" at end).
Any reason why the test is commented?